apps/vx-scan/backend/src/app_config.test.ts

```diff
@@ -313,3 +340,46 @@ test('ballot batching', async () => {
   expect(cvrs[3]._batchId).not.toEqual(batch3Id);
   expect(cvrs[5]._batchId).toEqual(batch3Id);
 });
+
+test('unconfiguring machine', async () => {
+  const { apiClient, mockAuth, mockUsb, interpreter, workspace } =
+    await createApp();
+  await configureApp(apiClient, mockUsb);
+
+  jest.spyOn(interpreter, 'unconfigure');
+  jest.spyOn(workspace, 'reset');
+
+  await apiClient.unconfigureElection({});
+
+  expect(interpreter.unconfigure).toHaveBeenCalledTimes(1);
+  expect(workspace.reset).toHaveBeenCalledTimes(1);
+  expect(mockAuth.clearElectionDefinition).toHaveBeenCalledTimes(1);
+  expect(mockAuth.clearPrecinctSelection).toHaveBeenCalledTimes(1);
+});
+
+test('auth', async () => {
+  const { apiClient, mockAuth, mockUsb } = await createApp();
+  await configureApp(apiClient, mockUsb);
+
+  await apiClient.getAuthStatus();
+  await apiClient.checkPin({ pin: '123456' });
+
+  void (await apiClient.readCardData({ schema: 'ScannerReportDataSchema' }));
+  void (await apiClient.writeCardData({
+    data: {},
+    schema: 'ScannerReportDataSchema',
+  }));
+
+  expect(mockAuth.getAuthStatus).toHaveBeenCalledTimes(1);
+  expect(mockAuth.checkPin).toHaveBeenCalledTimes(1);
+  expect(mockAuth.checkPin).toHaveBeenNthCalledWith(1, { pin: '123456' });
+  expect(mockAuth.readCardData).toHaveBeenCalledTimes(1);
+  expect(mockAuth.readCardData).toHaveBeenNthCalledWith(1, {
+    schema: ScannerReportDataSchema,
+  });
+  expect(mockAuth.writeCardData).toHaveBeenCalledTimes(1);
+  expect(mockAuth.writeCardData).toHaveBeenNthCalledWith(1, {
+    data: {},
+    schema: ScannerReportDataSchema,
+  });
```

Comment:
As simple as these auth tests are, these are what caught the Zod schema serialization error!